charts/tempo-operator/templates/operator-deployment.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    {{- include "tempo-operator.labels" . | nindent 4 }}
+  name: {{ include "tempo-operator.fullname" . }}-controller
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "tempo-operator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        {{- include "tempo-operator.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.manager }}
+      containers:
+        - name: manager
+          {{- if not .args }}
+          args:
+            - --zap-log-level=info
+            - start
+            - --config=controller_manager_config.yaml
+          {{- else -}}
+          {{- toYaml .args | nindent 12 }}
+          {{- end }}
+          env:
+          - name: RELATED_IMAGE_TEMPO
+            value: docker.io/grafana/tempo:2.5.0
```

Comment:
This image (and the other `RELATED_IMAGE` vars) change with (almost) every operator release, and must be in sync with the operator image.